README.md

```diff
@@ -74,10 +73,96 @@ supervisorctl status cosmian_kms
 ## Server parameters
 
 If a configuration file is provided, parameters are set following this order:
-- conf file (env variable `KMS_SERVER_CONF` set by default to `/etc/cosmian_kms/conf.toml`)
+- conf file (env variable `KMS_SERVER_CONF` set by default to `/etc/cosmian_kms/server.toml`)
 - default (set on struct)
 
 otherwise the parameters are set following this order:
 - args in command line
 - env var
 - default (set on struct)
+
+## Use the KMS inside a Cosmian VM on SEV/TDX
+
+See [this README](https://github.com/Cosmian/cosmian_vm) for more details about Cosmian VM. 
+
+To deploy the KMS inside a Cosmian VM, follow these steps:
+
+```console
+$ # Copy from resources/supervisor/cosmian_kms.ini
+$ sudo vi /etc/supervisord.d/cosmian_kms.ini
+$ # Copy the kms server binary
+$ sudo cp somelocation/cosmian_kms_server /usr/sbin/cosmian_kms
+$ # Create a conf file for the KMS (from resources/server.toml)
+$ sudo vi /etc/cosmian_kms/server.toml
+$ sudo supervisorctl reload 
+$ sudo supervisorctl start cosmian_kms
+$ # Check logs
+$ sudo tail -f /var/log/cosmian_kms.err.log
+$ sudo tail -f /var/log/cosmian_kms.out.log
+```
+
+Now you can interact with your KMS through the KMS CLI. 
+
+You can also interact with the Cosmiam VM Agent through its own CLI as follow:
+
+```console
+$ # From your own host
+$ # Snapshot the VM (it could take a while)
+$ ./cosmian_vm --url https://domain.name:port snapshot 
+$ # Sometimes, verify it
+$ ./cosmian_vm --url https://domain.name:port verify --snapshot ./cosmian_vm.snapshot
+Reading the snapshot...
+Fetching the collaterals...
+[ OK ] Verifying TPM attestation
+[ OK ] Verifying TEE attestation
```

Comment:
```suggestion
# From your own host
# Snapshot the VM (it could take a while)
$ ./cosmian_vm --url https://<DOMAIN_NAME>:<PORT> snapshot 

# Sometimes, verify it
$ ./cosmian_vm --url https://<DOMAIN_NAME>:<PORT> verify --snapshot ./cosmian_vm.snapshot
Reading the snapshot...
Fetching the collaterals...
[ OK ] Verifying TPM attestation
[ OK ] Verifying TEE attestation
```